math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/Regressor.scala

```diff
@@ -0,0 +1,49 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements. See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership. The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License. You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the License is distributed on an
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  * KIND, either express or implied. See the License for the
+  * specific language governing permissions and limitations
+  * under the License.
+  */
+
+package org.apache.mahout.math.algorithms.regression
+
+import org.apache.mahout.math.algorithms.regression.tests._
+import org.apache.mahout.math.drm._
+import org.apache.mahout.math.{Vector => MahoutVector}
+import org.apache.mahout.math.algorithms.Model
+import org.apache.mahout.math.drm.DrmLike
+
+import scala.reflect.ClassTag
+
+/**
+  * Abstract of Regressors
+  */
+trait Regressor[K] extends Model {
```

Comment:
+1 to `model` persistence.  Though we need some thinking here.  We could use persist/read the model as a set of Drms, since we have native-to-mahout support for this. ie. drm.dfsWrite(..), drmdfsRead(...). this can be quite clunky though.  (see NaiveBayes model.persist(...) eg.)